packages/core/src/codewhisperer/commands/startSecurityScan.ts

```diff
@@ -250,7 +250,11 @@ export async function startSecurityScan(
                 CodeScansState.instance.setMonthlyQuotaExceeded()
             }
         }
-        codeScanTelemetryEntry.reason = (error as SecurityScanError).message
+        if ((error as ToolkitError).code == 'ContentLengthError') {
+            codeScanTelemetryEntry.reason = 'Payload size limit reached'
```

Comment:
reasonDesc, not reason

Reply:
`Security scans` was using `reason` from the start, Let me discuss with @ctlai95 as this will impact metrics in kibana accordingly.

Reply:
Discussed and I will update `reason` with `reasonDesc` for security scans.